src/components/asset-list/RecyclerAssetList2/profile-header/ProfileRowWrapper.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from 'react';
+import { Box } from '@/design-system';
+import { ProfileStickyHeaderHeight } from './ProfileStickyHeader';
+
+export function ProfileRowWrapper({ children }: { children: React.ReactNode }) {
+  return (
+    <Box
+      alignItems="center"
+      width="full"
+      marginTop={{ custom: -ProfileStickyHeaderHeight }}
```

Comment:
🤔

Reply:
Yeah, it is a bit hacky, but I wasn't too sure on the best way of making dynamic sticky header visibility work well with our sticky logic in RAL.